Description: Disable tests using fontbakery as it is not yet available
Author: Balasankar C <balasankarc@debian.org>
Forwarded: not-needed
Last-Update: 2021-08-15
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Makefile
+++ b/Makefile
@@ -90,9 +90,9 @@
 
 test: ttf otf $(PDFS)
 	# fontbakery check-fontval $(BLDDIR)/*.ttf <- enable when https://github.com/microsoft/Font-Validator/issues/62 fixed
-	fontbakery check-ufo-sources $(SRCDIR)/*.ufo
-	fontbakery check-opentype $(BLDDIR)/*.otf
-	fontbakery check-googlefonts -x com.google.fonts/check/version_bump -x com.google.fonts/check/repo/zip_files -x com.google.fonts/check/metadata/escaped_strings -x com.google.fonts/check/family/vertical_metrics -x com.google.fonts/check/vertical_metrics_regressions -x com.google.fonts/check/fontdata_namecheck $(BLDDIR)/*.ttf
+	# fontbakery check-ufo-sources $(SRCDIR)/*.ufo
+	# fontbakery check-opentype $(BLDDIR)/*.otf
+	# fontbakery check-googlefonts -x com.google.fonts/check/version_bump -x com.google.fonts/check/repo/zip_files -x com.google.fonts/check/metadata/escaped_strings -x com.google.fonts/check/family/vertical_metrics -x com.google.fonts/check/vertical_metrics_regressions -x com.google.fonts/check/fontdata_namecheck $(BLDDIR)/*.ttf
 
 clean:
 	@rm -rf $(BLDDIR)
